dubbo-registry/dubbo-registry-default/src/test/java/org/apache/dubbo/registry/dubbo/RegistryDirectoryTest.java

```diff
@@ -1014,6 +1014,37 @@ public void test_Notified_acceptProtocol2() {
         List<Invoker<DemoService>> invokers = registryDirectory.list(invocation);
         Assert.assertEquals(2, invokers.size());
     }
+    
+    @Test
+    public void test_Notified_withGroupFilter() {
+    	URL directoryUrl = noMeaningUrl.addParameterAndEncoded(Constants.REFER_KEY, "interface" + service + "&group=group1,group2");
+    	RegistryDirectory directory = this.getRegistryDirectory(directoryUrl);
+    	URL provider1 = URL.valueOf("dubbo://10.134.108.1:20880?methods=getXXX&group=group1&mock=false");
+    	URL provider2 = URL.valueOf("dubbo://10.134.108.1:20880?methods=getXXX&group=group2&mock=false");
+    	
+    	List<URL> providers = new ArrayList<>();
+    	providers.add(provider1);
+    	providers.add(provider2);
+    	directory.notify(providers); 
+    	
+    	invocation = new RpcInvocation();
+    	invocation.setMethodName("getXXX");

```

Comment:
might be method name can be define as a variable which can be used in the url plus here as well.